src/modules/tokens/services/token.compute.service.ts

```diff
@@ -232,7 +232,7 @@ export class TokenComputeService implements ITokenComputeService {
             metric: 'priceUSD',
         });
 
-        return values24h[0]?.value ?? undefined;
```

Comment:
Should we include this fix? It seems to be unrelated to the issue we were seeing but rather related to handling the case of tokens launched in the last day. Maybe we want to properly address this later?

Reply:
done